src/main/java/duke/Main.java

```diff
@@ -0,0 +1,329 @@
+                printLine();
+            }
+        } else {
+            try {
+                addTask(words[0], tasks, command);
+            } catch (WrongCommandException e) {
+                printLine();
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                printLine();
+            } catch (EmptyTodoException e) {
+                System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+            } catch (EmptyDeadlineException e) {
+                System.out.println("☹ OOPS!!! The description of a deadline cannot be empty.");
+            } catch (NoDeadlineTimeMarkerException e) {
+                System.out.println("☹ OOPS!!! You should mark the time for a deadline with \"/by\"");
+            } catch (NoDeadlineTimeException e) {
+                System.out.println("☹ OOPS!!! You should enter a time for deadline.");
+            } catch (EmptyEventException e) {
+                System.out.println("☹ OOPS!!! The description of a event cannot be empty.");
+            } catch (NoEventTimeMakerException e) {
+                System.out.println("☹ OOPS!!! You should mark the time for an event with \"/at\"");
+            } catch (NoEventTimeException e) {
+                System.out.println("☹ OOPS!!! You should enter a time for event.");
+            } catch (InvalidCommandException e) {
+                System.out.println("unknown error happens.");
+                //this is not reachable as all Invalid Commands are dealt above
+            }
+        }
+
+    }
+
+    private static void printExitMessage() {
+        printLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        printLine();
+    }
+
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void printGreetingMessage() {
+        printLine();
+
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        System.out.println("What can I do for you?");
+    }
+
+    private static void addTask(String beginning, ArrayList<Task> tasks, String command) throws InvalidCommandException {
+
+        int dividerPosition;
+
+
+        switch (beginning) {
+            case ADD_EVENT:
+                dividerPosition = command.indexOf("/at");
+
+                if (command.substring(5).isBlank()) {
+                    throw new EmptyEventException();
+                }
+                if (dividerPosition == -1) {
+                    throw new NoEventTimeMakerException();
+                }
+                if (command.substring(5, dividerPosition).isBlank()) {
+                    throw new EmptyEventException();
+                }
```

Comment:
Perhaps you can write some comments on what these conditions do to improve readability?